lib/partition_setup.pm

```diff
@@ -254,9 +254,11 @@ sub addlv {
     send_key $cmd{system_view};
     send_key 'home';
     send_key_until_needlematch('volume_management_feature', 'down');
+    wait_still_screen 2;
```

Comment:
Just for the notice, it's sad that we have to use these hacks. Fedora guys also use it quite often in their code: https://pagure.io/fedora-qa/os-autoinst-distri-fedora/blob/master/f/tests/_software_selection.pm =(

Reply:
Agreed, does not look nice and also the functional part is questionable at some point. Nevertheless I hope we can reduce failures a bit at least. It is quite a pity that send_key does not return any value, so we can determine if the pressed key has been accepted or not by SUT